WordPress/src/main/java/org/wordpress/android/ui/stats/StatsGeoviewsFragment.java

```diff
@@ -20,6 +31,121 @@
 public class StatsGeoviewsFragment extends StatsAbstractListFragment {
     public static final String TAG = StatsGeoviewsFragment.class.getSimpleName();
 
+
+    private void hideMap() {
+        if (!isAdded()) {
+            return;
+        }
+
+        mTopPagerContainer.setVisibility(View.GONE);
+    }
+
+    @SuppressLint("NewApi")
+    private void showMap(final List<GeoviewModel> countries) {
+        if (!isAdded()) {
+            return;
+        }
+
+        // removeOnGlobalLayoutListener is available on >= JELLY_BEAN. The map is hidden on older devices.
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
You can always show the map by using the similarly-named [removeGlobalOnLayoutListener](http://developer.android.com/reference/android/view/ViewTreeObserver.html#removeGlobalOnLayoutListener(android.view.ViewTreeObserver.OnGlobalLayoutListener)) which has been available since API 1. On Jelly Bean and later this simply calls `removeOnGlobalLayoutListener`.